HHS/OPRE-OPS
# refactor: ♻️ updates logic for disabled icons text

## What changed

 - updates logic for disabled icons text

## Issue

#1430 #1470 

## How to test

1. find a non-DRAFT/PLANNED budget line from Agreement BLI Tab or BudgetLines list
2. hover over row and attempt to delete
3. see new tooltip
4. 💰 profit

## Screenshots

<img width="1104" alt="image" src="https://github.com/user-attachments/assets/2245b31e-a36c-4c72-811e-f192c1cf4762">

https://github.com/user-attachments/assets/cc6feabf-331a-40c8-8d3a-92cab2059e79

## Definition of Done Checklist
